app/models/user.rb

```diff
@@ -86,6 +86,14 @@ def active_profile
     profiles.find(&:active?)
   end
 
+  # This user's most recently activated profile that has also been deactivated
+  # due to a password reset, or nil if there is no such profile
+  # @return [Profile,nil]
```

Comment:
I don't think these comments are necessary (to be fair, I feel this way about almost all code comments) :)